abilitybot/configureBotExample.js

```diff
@@ -0,0 +1,91 @@
+import { CharInfo } from "../bossroom";
+
+let charType = 0; // Healer
+let rg = null;
+
+
+/**
+ * Defines the type of character that the game should use for this bot.
+ */
+export function getCharacterType() {
+    return CharInfo.type[charType]; 
+}
+
+/**
+ * One of ...
+ * MANAGED - Server disconnects/ends bot on match/game-scene teardown
+ * PERSISTENT - Bot is responsible for disconnecting / ending itself
+ */
+export function getBotLifecycle() {
+return 'PERSISTENT';
+}
+
+/**
+ * @returns {boolean} true if I'm an in-game character, or false if I'm an invisible navigator/observer/etc.
+ */
+export function isSpawnable() {
+    return true;
+}
+
+export function isComplete() {
+    return rg ? rg.isComplete() : false;
+}
+
+export async function configureBot(rgObject) {
+    rg = rgObject;
+    rg.automatedTestMode = true;
+
+    // validate that we're in the game
+    await rg.waitForScene("BossRoom");
+
+    // find the closest human player and use a heal ability on them
+    let target = await rg.findNearestEntity("HumanPlayer");
```

Comment:
Doesn't findNearestEntity already fail if it doesn't exist? So the second line may be redundant?

Reply:
Today we don't stop the bot when a validation fails. For test bots I'd love this behavior, but dynamic bots shouldn't stop if one of their evaluations fail.